yaks/spec/unit/yaks/mapper/attribute_spec.rb

```diff
@@ -70,11 +72,41 @@
   end
 
   describe "#add_to_resource" do
+
+    let(:options) { {if: ->{ true }} }
+    let(:options_false) { {if: ->{ 0 == 1 }} }
```

Comment:
It checks that the proc is `call`'ed but not `instance_eval`'d. What I mean is to have something like:

```ruby
let(:options_false) { {if: ->{ object.name.length > 10 }} }
```